WalletWasabi/WabiSabi/Client/CoinJoinManager.cs

```diff
@@ -17,16 +17,16 @@ namespace WalletWasabi.WabiSabi.Client
 {
 	public class CoinJoinManager : BackgroundService
 	{
-		public CoinJoinManager(WalletManager walletManager, RoundStateUpdater roundStatusUpdater, HttpClientFactory backendHttpClientFactory, ServiceConfiguration serviceConfiguration)
+		public CoinJoinManager(WalletManager walletManager, RoundStateUpdater roundStatusUpdater, IBackendHttpClientFactory backendHttpClientFactory, ServiceConfiguration serviceConfiguration)
 		{
 			WalletManager = walletManager;
-			ArenaRequestHandler = new WabiSabiHttpApiClient(backendHttpClientFactory.NewBackendHttpClient(Mode.SingleCircuitPerLifetime));
+			HttpClientFactory = backendHttpClientFactory;
 			RoundStatusUpdater = roundStatusUpdater;
 			ServiceConfiguration = serviceConfiguration;
 		}
 
 		public WalletManager WalletManager { get; }
-		public IWabiSabiApiRequestHandler ArenaRequestHandler { get; }
+		public IBackendHttpClientFactory HttpClientFactory { get; }
```

Comment:
As mentioned here https://github.com/zkSNACKs/WalletWasabi/pull/6404/files#r711385325 maybe it would be good to write:

`public IHttpClientFactory BackendHttpClientFactory { get; }`